app/src/components/map/olMapHelpers.js

```diff
@@ -0,0 +1,349 @@
+import TileLayer from 'ol/layer/Tile';
+import XYZSource from 'ol/source/XYZ';
+import Cluster from 'ol/source/Cluster';
+import VectorSource from 'ol/source/Vector';
+import { LineString, Point, Polygon } from 'ol/geom';
+import { Vector as VectorLayer } from 'ol/layer';
+import { createEmpty, extend, getWidth } from 'ol/extent';
+import monotoneChainConvexHull from 'monotone-chain-convex-hull';
+import store from '@/store';
+
+import { Feature } from 'ol';
+import { fromLonLat } from 'ol/proj';
+import {
+  Circle as CircleStyle,
+  Fill,
+  Stroke,
+  Style,
+  Text,
+} from 'ol/style';
+import { indicatorClassesUnicode } from '../../config/trilateral';
+import { getColor } from './olMapColors';
+
+/**
+ * generate a layer from a given config Object
+ * @returns {*} returns ol layer
+ */
+// eslint-disable-next-line import/prefer-default-export
+export function createLayerFromConfig(config) {
+  let source;
+  if (config.protocol === 'xyz') {
+    source = new XYZSource({
+      attributions: config.attribution,
+      crossOrigin: 'anonymous',
+      url: config.url,
+    });
+  }
+  return new TileLayer({
+    name: config.name,
+    visible: config.visible,
+    maxZoom: config.maxNativeZoom,
+    source,
+  });
+}
+
+const circleDistanceMultiplier = 1;
+const circleFootSeparation = 28;
+const circleStartAngle = Math.PI / 2;
+
+const convexHullFill = new Fill({
```

Comment:
Here it would be good to use the default colors as defined in https://github.com/w3geo/eodash/blob/ol-clustering/app/src/components/Map.vue#L233-L240

Reply:
of course, i fixed this with [9c13827](https://github.com/eurodatacube/eodash/pull/1204/commits/9c138278c614e4874a1c794388f4491fb2178f3e).